l10n_br_account_banking_payment_cnab/febraban/cnab_240/cnab_240.py

```diff
@@ -49,9 +52,23 @@ def get_bank(bank):
         elif bank == '104':
             from bancos.cef import Cef240
             return Cef240
+        elif bank == '033':
+            from bancos.santander import Santander240
+            return Santander240
         else:
             return Cnab240
 
+    @staticmethod
+    def get_cnab_bank(bank):
+        if bank == '341':
+            from cnab240.bancos import itau
+            return itau
+        elif bank == '033':
+            from cnab240.bancos import santander
+            return santander
+        else:
+            raise NotImplementedError
+
```

Comment:
This sequence of `if ... elif` should also be replaced by a module level dictionary.

Ideally, this dictionary should belong to the `cnab240` module itself, but until there is such a dictionary there, a dictionary the level of this module would be better than this list of `if ... elif`